packages/cli/workspace/oss-validator/src/rules/no-duplicate-overrides/__test__/no-duplicate-overrides.test.ts

```diff
@@ -0,0 +1,30 @@
+import { AbsoluteFilePath, RelativeFilePath, join } from "@fern-api/fs-utils";
+
+import { ValidationViolation } from "../../../ValidationViolation";
+import { getViolationsForRule } from "../../../testing-utils/getViolationsForRule";
+import { NoDuplicateOverridesRule } from "../no-duplicate-overrides";
+
+describe("compatible-ir-versions", () => {
+    it("simple failure", async () => {
+        const violations = await getViolationsForRule({
+            rule: NoDuplicateOverridesRule,
+            absolutePathToWorkspace: join(
+                AbsoluteFilePath.of(__dirname),
+                RelativeFilePath.of("fixtures"),
+                RelativeFilePath.of("simple")
+            ),
+            cliVersion: "0.1.3-rc0"
+        });
+
+        const expectedViolations: ValidationViolation[] = [
+            {
+                severity: "error",
+                relativeFilepath: RelativeFilePath.of("openapi/openapi.yml"),
+                nodePath: ["paths", "/a/b", "get"],
+                message: "Duplicate SDK method found: group 'a' with method 'b'"
```

Comment:
It might be a bit cleaner to do `Encountered duplicate x-fern-sdk-method-name users.create`

Reply:
`Type {name of type} already exists`
`SDK method users.get already exists (x-fern-sdk-group-name: users, x-fern-sdk-method-name: get)`